### PR TITLE
Added Julia Language

### PR DESCRIPTION
**Pull request by kshyatt**

### CODE REVIEW

**Comment by tkelman** [comment]
+1. We also have a separate "up for grabs" label but if we have to pick one I think the "intro issue" would be the better one to advertise.

File: _data/projects/julia.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name: Julia Language
+desc: Julia is a high-level, high-performance dynamic programming language for technical computing. https://www.julialang.org
+site: https://github.com/JuliaLang/julia
```
Comment (shiftkey) [inline _data/projects/julia.yml:3]: Entirely up to you whether you'd like this to point to the repository or the top-level site, but this is the "project name" link the user will click.

File: _data/projects/julia.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+name: Julia Language
+desc: Julia is a high-level, high-performance dynamic programming language for technical computing. https://www.julialang.org
```
Comment (shiftkey) [inline _data/projects/julia.yml]: We don't render the links currently (see #312 for us supporting markdown) but using `https` here triggers a certificate error (as it expects julialang.org but gets github.com)

**Comment by kshyatt** [comment]
Hi @shiftkey, I've updated the PR. Hopefully this is ok now? Thanks for your tips!

**Comment by shiftkey** [comment]
@kshyatt looks great to me - thanks!

**Pull request merged**